src/PolykeyClient.ts

```diff
@@ -13,6 +14,21 @@ import * as events from './events';
 import config from './config';
 import * as networkUtils from './network/utils';
 import clientClientManifest from './client/callers';
+import {option} from "fast-check";
+
+/**
+ * Optional configuration for`PolykeyClient`.
+ */
+type PolykeyClientOptions = {
+  nodePath?: string;
+  tokenBase: string;
+  // wsConfig?: Partial<WebSocketConfig>;
+  rpc: {
+    callTimeoutTime: number;
+    parserBufferSize: number;
+  };
+  wsConfig?: Partial<WebSocketConfig>;
+};
```

Comment:
I think generally if an option is not defaulted then it can be  and `thing?: Type` optional type here.

Also @CMCDragonkai do we want this to be flatter? As i the `rpc` and `wsConfig` options flattened into the `PolykeyClientOptions`?

Also, remove the commented out `wsConfig` line.

Reply:
Done

Reply:
Subdomains will all be grouped.

Because websocket is just encapsulated as part of PKClient, and it's not a subdomain, I might just flatten it.